### PR TITLE
Yavyx/update installation guide

### PR DESCRIPTION
**Pull request by yavyx**
Added Ubuntu-specific instructions to install Docker.

### CODE REVIEW

File: docs/install-docker.md [hunk 1]
```diff
@@ -52,7 +55,41 @@ sudo systemctl start docker
 sudo usermod -aG docker $(whoami)
 ```
 
-Note that CanDIG requires **Docker Compose v2**, which is provided alongside the latest version of Docker. Versions of Docker which do not provide Docker Compose will unfortunately not work with CanDIG.
```
Comment (mshadbolt) [inline docs/install-docker.md:55]: Is this note no longer relevant?

Comment (yavyx) [reply docs/install-docker.md:55]: It's still there, I moved it to the top section where it makes more sense.

Comment (mshadbolt) [reply docs/install-docker.md:55]: oh right sorry I missed that!

**Review by mshadbolt — APPROVED** [review]
Looks good to me!

**Pull request merged**